targets/targets.json

```diff
@@ -3693,6 +3693,41 @@
         "release_versions": ["2", "5"],
         "device_name": "STM32F469NI",
         "bootloader_supported": true
+    },
+	"SDP-K1": {
+        "inherits": ["FAMILY_STM32"],
+        "core": "Cortex-M4",
+		"supported_form_factors": ["ARDUINO"],
+        "extra_labels_add": [
+            "STM32F4",
+            "STM32F469",
+            "STM32F469NI",
+            "STM32F469xI",
+            "STM32F469xx"
+        ],
+		"config": {
+            "clock_source": {
+                "help": "Clock source to use, can be XTAL or RC",
+                "value": "USE_PLL_HSE_XTAL|USE_PLL_HSI",
+                "macro_name": "CLOCK_SOURCE"
+            },
+			"clock_freq": {
+				"help": "Clock frequency in Mhz",
+				"value": "8",
+				"macro_name": "CLOCK_FREQUENCY_MHZ"
+			}
+        },
+		"overrides": {
+            "lse_available": 0
+        },
+		"macros_add": ["USB_STM_HAL", "USBHOST_OTHER"],
+		"device_has_add": [
+            "ANALOGOUT",
+			"MPU"
+        ],
+        "device_name": "STM32F469NI",
+		"release_versions": ["5"],
```

Comment:
fix alignment here on this line

Reply:
https://github.com/malavikasajikumar/mbed-os/commit/a8fc16e6de6623bd21753c83d113355c1b67fb4f